org.mixedrealitytoolkit.spatialmanipulation/Solvers/HandConstraintPalmUp.cs

```diff
@@ -274,21 +274,47 @@ private bool IsUserGazeMeetingThresholdRequirements(XRNode hand)
                 Ray? gazeRay = null;
                 bool usedEyeGaze = false;
 
-                if (ControllerLookup != null &&
-                    ControllerLookup.GazeController != null &&
-                    (ControllerLookup.GazeController.currentControllerState.inputTrackingState &
-                    (InputTrackingState.Position | InputTrackingState.Rotation)) > 0)
+                if (ControllerLookup != null) //Note: Will be removed for XRI3 migration completion
```

Comment:
I think you were right earlier :)  we need branching for the old rig to work.   I think this is a case where you can't remove this.  Surround with `#pragma warning disable`

```cs
#pragma warning disable CS0618 // Type or member is obsolete
if (ControllerLookup != null)
{
    // your code
}
#pragma warning restore CS0618
```

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/749/commits/2e7783a6407a5447c9ecefb943a266855b096514 , resolving.